### DISCUSSION TITLE
global block limitation?

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1316] [Q&A]
Is my understanding correct that DNN modules can't be saved into a global block?

**Comment by mksmanish79** [comment]
You can use DNN modules in the global block but make sure after adding the module set module Display app on all pages to true.

**Comment by meetmandeep** [comment]
To illustrate, access app settings from: 

![image](https://user-images.githubusercontent.com/29958123/119900340-42b19300-bef9-11eb-8c13-8f8b24fbe1c9.png)

and display app on all pages from advanced tab:

![image](https://user-images.githubusercontent.com/29958123/119900464-68d73300-bef9-11eb-8b2a-b1f92e8a8d86.png)

**Comment by eswanzey** [comment]
It makes me wonder whether 'display on all pages' means something different if set within DNN, or set within Vanjaro.

.putting it on the punch list
